api4/post.go

```diff
@@ -109,12 +112,20 @@ func getPostsForChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	afterPost := r.URL.Query().Get("after")
+	if len(afterPost) > 0 && len(afterPost) != 26 {
```

Comment:
Should we be using `model.IsValidId` here when non-empty?

Reply:
Thanks, will do that.

Reply:
Done.